moduleroot/.travis.yml.erb

```diff
@@ -106,11 +106,41 @@ matrix:
       <%= key %>: <%= job[key].gsub(/@@SET@@/, set['set']).gsub(/@@COLLECTION@@/, set.fetch('collection', 'puppet6')).gsub(/@@TESTMODE@@/, set.fetch('testmode', 'apply')) %>
 <%   end -%>
 <% end -%>
+<% if @configs['use_litmus'] == true -%>
+<%
+  @configs.delete('use_litmus')
+  # add default values
+  @configs.merge!('use_litmus'=> {'puppet_collection' => ['puppet5', 'puppet6'], 'provision_list' => ['travis_el','travis_deb'], 'rvm' => '2.5.3'})
+  end
```

Comment:
@lionce and I chatted about this bit and came up with a better solution to be implemented after the holidays:

```
@config['use_litmus'] = {} if @config['use_litmus']  == true
```

Reply:
+1 to @scotje comment.  use_litmus is boolean-y.  Litmus configuration should be a different name.

Reply:
Hey @DavidS, 
To remove duplicated code in case of use_litmus == true, first i deleted its value and added and empty one  (This was the only way of assigning it an empty value). Also updated Readme file.

Cheers!

Reply:
Hey @glennsarti , @scotje 

I'll add new hash for litmus specific configurations and keep use_litmus as a boolean value
Something like:
```
use_litmus: true | false
litmus:
    provision_list: [list1, list2]
    puppet_collection: [puppet5, puppet6]
    rvm: 2.5.3
    install_wget: yes | no
```
Cheers!